build/pr-pipeline.yml

```diff
@@ -167,7 +167,7 @@ stages:
       resourceGroup: $(ResourceGroupName)
       sqlServerName: $(DeploymentEnvironmentName)
       schemaAutomaticUpdatesEnabled: 'auto'
-      sqlServerAdminPassword: $[stageDependencies.UpdateRandom.Password.outputs['SetVariablesFromRandomString.password']]
+      userAssignedManagedIdentityName: "$(DeploymentEnvironmentName)-uami"
```

Comment:
Looks like the AppService template is failing for Cosmos sites, perhaps all the deployments need to "dependsOn": deploySqlServer so that the UAMI is created first